Calendar/js/app.js

```diff
@@ -0,0 +1,233 @@
+    for (var i = startYear; i <= endYear; i++){
+        _selectedYearElement.innerHTML += '<option value="' + i + '">' + i + '</option>';
+    }
+
+    for (var i = 0; i < 12; i++) {
+        _selectedMonthElement.innerHTML += '<option value="' + i + '">' + _months[i].toUpperCase() + '</option>';
+    }
+}
+
+function generateWeekDays() {
+    var weekHeader = document.getElementsByClassName('week-days')[0];
+    var dayNames = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+    for(var i = 0; i < 7; i++) {
+        weekHeader.innerHTML += '<div class="day">' + dayNames[i] + '</div>';
+    }
+}
+
+function calculateWeekNo() {
+    var firstDateOfMonth = new Date(_year, _month, 1);
+    var firstDateOfYear = new Date(_year, 0, 1);
+    if (firstDateOfMonth.getDay() == 0) {
+        firstDateOfMonth = new Date(_year, _month, 2);
+    }
+    return (Math.floor((firstDateOfMonth.getTime() - firstDateOfYear.getTime()) / (1000 * 60 * 60 * 24 * 7)) + 1);
+}
+
+//Event Listeners
+document.getElementById('prev-click').addEventListener('click', function() {
+    if (_month == 0) {
+        if (_year == _startYear) {
+            alert('You can not see previous year');
+            return ;
+        }
+        _year -= 1;
+        _month = 11;
+    }
+    else {
+        _month -= 1;
+    }
+    createCalendar();
+});
+
+document.getElementById('next-click').addEventListener('click', function() {
+    if (_month == 11) {
+        if (_year == _endYear) {
+            alert('You can not see next year');
+            return ;
+        }
+        _year += 1;
+        _month = 0;
+    }
+    else{
+        _month += 1;
+    }
+    createCalendar();
+});
+
+_selectedYearElement.addEventListener('change', function() {
+    _year = Number(_selectedYearElement.value);
+    createCalendar();
+});
+
+_selectedMonthElement.addEventListener('change', function() {
+    _month = Number(_selectedMonthElement.value);
+    createCalendar();
+});
+
+function generateBirthdays(lastDate) {
+    var birthdaysOnDate;
+    var currentDateElement;
+    var birthdayElement;
+    var birthdayBlockInnerHTML;
+    console.log('check');
+
+    for (var i = 1; i <= lastDate; i++) {
+        birthdaysOnDate = birthdays[String(_month + 1)][String(i)];
+        console.log(birthdaysOnDate);
+        birthdayBlockInnerHTML = '';
+        if (typeof(birthdaysOnDate) != "undefined") {
+            currentDateElement = document.querySelector('[data-month-date="' + (_month + 1) + '-' + i + '"]');
+            for (var j = 0; j < birthdaysOnDate.length; j++) {
+                birthdayBlockInnerHTML += '<p>' + birthdaysOnDate[j] + '</p>';
+                console.log(j);
+            }
+            currentDateElement.innerHTML = '<div id="bb" class="birthdayBlock">' + birthdayBlockInnerHTML + '</div>';
+            currentDateElement.innerHTML += i + ' <i class="fa fa-birthday-cake"></i>';
+            currentDateElement.style.color = "#F96120" //"#A5DE60"; //"#D01315"
```

Comment:
make a class for this. as we discussed yesterday

Reply:
I changed the current date styling with a new class but forgot to do this. I will update the code asap.